### PR TITLE
create Observation.derivedFrom Resource example

### PR DESCRIPTION
**Pull request by jamais-vu**
Example: using `Observation.derivedFrom` for linking `z-score` and `% Pred` to their respective FVC Observation.

### CODE REVIEW

File: FVC-Observation-derivedFrom-example.js
Comment (jdjkelly) [inline FVC-Observation-derivedFrom-example.js:54]: Yeah and we discussed just skipping this for the spiro values today

File: FVC-Observation-derivedFrom-example.js
Comment (jdjkelly) [inline FVC-Observation-derivedFrom-example.js:120]: No not the same. 

Three things:

1. You can look them up when you know the code, for ex: https://loinc.org/19868-9/ - just to see what they mean etc
2. You can go to the other codebase - pft-extractor and reference them by the lookup mapping I've done previously (I think I showed you this?)
3. Richard (RIP, he died last year) was this guy who maintained a blog about PFTs and he posted his own LOINC mapping as an excel file in this post: https://www.pftforum.com/blog/loinc-and-why-it-matters-to-your-his-interface/

File: FVC-Observation-derivedFrom-example.js
Comment (jdjkelly) [inline FVC-Observation-derivedFrom-example.js:142]: Eh, sure. 

My opinion is that as a data type they should be float of min 0 max 1 and then the % thing is just a presentation question. Given with what we're working with here... this seems not horrible and okay.

File: FVC-Observation-derivedFrom-example.js
Comment (jdjkelly) [inline FVC-Observation-derivedFrom-example.js:143]: Yeah maybe this is GLI or etc - I can show you some example PFTs where this data is provided (in addition to the standard one).

**Review by jdjkelly — APPROVED** [review]

**Comment by jdjkelly** [comment]
@jamais-vu so typical workflow for a github setup is:

1. Require automatic/continuous integration checks to be green (there are none set up on this repo - so it's green - an example of one would be running ESLint on every branch and seeing if the "linter" rules for the typescript/javascript pass or not

2. Require a review to be completed with an approval (that's done)

Then after that there may be merge timing rules etc but generally it's always the implementer who clicks the button. I've configured this repo to *require* the approval (2) just to demo this for you. So for subsequent PRs when you open it the merge option won't be available until after review.

Anyways, you can push button

File: FVC-Observation-derivedFrom-example.js
Comment (jamais-vu) [reply FVC-Observation-derivedFrom-example.js:142]: Using float and formatting to percent in presentation layer makes sense, I'll do that moving forward.

File: FVC-Observation-derivedFrom-example.js
Comment (jamais-vu) [reply FVC-Observation-derivedFrom-example.js:120]: Thanks, I was having trouble finding them on LOINC but the Excel sheet solved that right away.

**Pull request merged**
